Makefile

```diff
@@ -78,13 +78,23 @@ help: ## Display this help message.
 	cd web; npm run build
 	cd web; npm run dev
 
-4_deploy_realm: ## Deploy GnoChess realm on local node.
+4_deploy_chess_realm: ## Deploy GnoChess realm on local node.
```

Comment:
i think we should just keep a single `4_deploy_realm` rule that uploads as many realm as needed (two at the moment).

Reply:
Yeah. makes sense. Done.